testing/integrations_test.go

```diff
@@ -174,6 +174,51 @@ func TestPrivateRedmineIntegration(t *testing.T) {
 	}
 }
 
+func TestPublicAzureIntegration(t *testing.T) {
```

Comment:
The integration test doesn't handle the line "// TODO 9: An issue in CLOSED column" in the testing scenarios as closed issue error.